src/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title><%= htmlWebpackPlugin.options.title %></title>
+  <link rel="stylesheet" href="./style.css">
+</head>
+<body class="body">
+  <span id="date-time">date-time goes here</span>
+  <h1>Leaderboard</h1>
+
+  <div class="list-score-section">
+    <div class="list-heading">
+      <h2>Recent Scores</h2>
+      <span>Refresh</span>
+    </div>
+    
+    <ul id="score-list">
+      <!-- Scores gos here -->
+    </ul>
+  </div>
+
+  <div class="add-score-section">
+    <h2>Add your score</h2>
+    <form action="">
+      <label for="name" hidden>Name:</label>
+      <input type="text" id="name" placeholder="Your name" />
+      <label for="score" hidden>Score:</label>
+      <input type="text" id="score" placeholder="Your score" />
+      <button type="submit" id="add-btn" class="add-btn">Submit</button>
+    </form>
+  </div>
+  <span id="message"></span>
+</body>
+<!-- <script src="./index.js" type="module"></script> -->
+</html>
```

Comment:
As per the project requirements, your design should match the wireframe design provided. You don't need to go crazy with the styling, but the bare minimum should be matched with the wireframe design.

| Your design  | Wireframe design |
| ------------- | ------------- |
| ![image](https://github.com/anyars-encarta/Leaderboard-List-App/assets/33268927/60240e24-09d3-4354-935e-52a722575c55) | ![image](https://github.com/anyars-encarta/Leaderboard-List-App/assets/33268927/3e30cb5f-f3d3-4a44-93ad-a875342ea526) |

Reply:
Thank you very much for the CR. I will definitely do that